apps/home/templates/authenticated-home.html

```diff
@@ -1,46 +1,124 @@
 {% extends "base.html" %}
 {% load i18n %}
 {% load static %}
+
+{% block bannerBackButton %}{% endblock %}
+
+{% block bannerTitle %}
+Developer Dashboard
+{% endblock %}
+
+{% block bannerDescription %}
+The Developer Sandbox lets you register applications to get credentials to access our synthetic (or sample) beneficiary data. You can use that data to build and test your application to get it ready for production!
+{% endblock %}
+
+{% block bannerCallToActionButtons %}
+<a class="cta-button ds-u-margin-y--1" href="{% url "oauth2_provider:register" %}" aria-label="Add an Application" target="_self">
+  Add an Application
+  <i data-feather="arrow-right"></i>
+</a>
+   
+<a class="cta-button ds-u-margin-y--1" href="https://bluebutton.cms.gov/developers/" aria-label="Read Our Documentation" target="_self">
+  Read our Documentation
+  <i data-feather="arrow-right"></i>
+</a>
+{% endblock %}
+
 {% block Content %}
 
 <div class="container">
-
   <div class="ds-l-row ds-u-justify-content--center ds-u-margin-y--4">
     <div class="ds-l-lg-col--11 ds-l-md-col--11 ds-l-sm-col--12">
-      <!-- Breadcrumbs -->
-      <ol class="breadcrumb">
-        <li class="active"><a href="/">Home</a></li>
-      </ol>
-
-      <!-- <h1>Dashboard</h1> -->
-
-      <!-- Welcome Message -->
-      <div class="ds-c-alert ds-u-margin-y--2">
-        <div class="ds-c-alert__body">
-          <h2 class="ds-c-alert__heading">Welcome Developers</h2>
-          <p class="ds-c-alert__text"> From here you can add, edit, and delete applications you wish to connect. You may also revoke access to any access tokens you have authorized.</p>
-        </div>
-      </div>
-
       {% include "include/messages.html" %}
 
-      <!-- Action Buttons -->
-      <div class="ds-u-display--flex ds-u-justify-content--center">
-        <a class="ds-c-button ds-c-button--default ds-u-margin--2" href="{% url 'oauth2_provider:authorized-token-list' %}">
-          Application Tokens
+      <h2>My Sandbox Apps</h2>
+
+      {% if applications %}
+      <div class="bb-app-card-container-2-2 ds-u-margin-y--3">
```

Comment:
The behaviour you're trying to describe here is "Explain to the user why they should register an application if they haven't already" so bringing the `if` down to this line and having it check for the lack of applications will better communicate this: `if not applications` containing what's currently on lines 76-82.

This also allows you to remove the else, the `for` loop will simply not execute since there is nothing to loop over. (I'm making an assumption here, but I'm hopeful)

Reply:
I like it. Added commit to fix this.